app/controllers/solidus_square/payments_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module SolidusSquare
+  class PaymentsController < BaseController
```

Comment:
This is an API endpoint, it should be under the API namespace, as the [Api::CheckoutController](https://github.com/nebulab/solidus_square/blob/main/app/controllers/solidus_square/api/checkout_controller.rb)